### PR TITLE
Simple samples

### PR DESCRIPTION
**Pull request by sjulti**
## Solving issue:
- uh_upload script is too complex

## Description
Smaller upload and download samples

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected or change in the architecture)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation and architecture diagram accordingly.

## What reviewers should especially pay attention to (if applicable):
*

### CODE REVIEW

**Review by kbelova — DISMISSED** [review]
LGTM

File: scripts/uh-upload/simple_download.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+import boto3
+import os
+import sys
+
+if __name__ == "__main__":
+    if len(sys.argv) < 2:
+        print("usage: simple_download.py <endpoint> [file [file ...]]")
```
[the remark this answers is not in the document]
Comment (sjulti) [reply scripts/uh-upload/simple_download.py]: No, we can run from any location as long as the path is either relative to the current working directory or absolute. This is a very common notation of giving files on the command line.

File: scripts/uh-upload/simple_upload.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    # all data will be stored under
+    target_bucket_name = "bucket"
+    target_s3.create_bucket(Bucket=target_bucket_name)
+
+    # upload each file from command line and output real and effective size
+    for id in range(2, len(sys.argv)):
+        file = sys.argv[id]
+
+        with open(file, 'rb') as f:
+            response = target_s3.put_object(Bucket=target_bucket_name, Key=os.path.basename(file), Body=f)
```
[the remark this answers is not in the document]
Comment (sjulti) [reply scripts/uh-upload/simple_upload.py]: This is purely using put_object, no multipart at all.

**Review by kbelova — APPROVED** [review]
LGTM

**Pull request merged**